.github/workflows/release-app.yml

```diff
@@ -148,6 +148,11 @@ jobs:
             pre-build-args: ""
             tauri-args: "--target x86_64-pc-windows-msvc --config src-tauri/tauri.conf.prod.json"
             os_type: "windows"
+          - platform: "ubuntu-latest"
+            args: "--target x86_64-unknown-linux-gnu --features metal"
```

Comment:
i dont think linux use metal 

what about mkl?

Reply:
@louis030195 oversight, added mkl and installed mkl lib now.